src/main/resources/application.properties

```diff
@@ -3,6 +3,10 @@ kafka.bootstrap.servers=localhost:9092
 
 quarkus.http.port=8085
 
+# Change port for tests to avoid messing with local Kafka instance
+%test.quarkus.http.port=9085
```

Comment:
I am not even sure we need that, as testcontainers assign an ephemeral port here already